app/commands/testingCentres.js

```diff
@@ -33,9 +33,17 @@ function testingCentres(body, bot, chatId, stateUserMessage) {
 
       const data = prepareAnswer.prepareTestingResourceAnswer(stateResources);
 
-      return bot.sendMessage(chatId, data, {
+      return bot.sendMessage(chatId, data[0], {
```

Comment:
Use further a loop here for sending a message by fetching data from the array into small acceptable parts.
Do an empty return in the end.